index.html

```diff
@@ -14,6 +14,47 @@
     <title>Adel's to-do app</title>
   </head>
   <body class="bg-blue-100">
+    <div
+      id="login-popup"
+      class="fixed left-0 right-0 top-0 z-50 hidden h-full items-center justify-center overflow-y-auto overflow-x-hidden bg-black/50"
+    >
```

Comment:
Let's make the background more darker? Users are not suppose to access the todo app at this point so no need to show them the app in the background.